grafana-plugin/src/containers/EscalationVariants/parts/EscalationVariantsPopup.tsx

```diff
@@ -155,10 +155,10 @@ const EscalationVariantsPopup = observer((props: EscalationVariantsPopupProps) =
             onChange={setSchedulesSearchTerm}
           />
           <GTable
-            emptyText={store.scheduleStore.getSearchResult() ? 'No schedules found' : 'Loading...'}
+            emptyText={store.scheduleStore.getSearchResult().results ? 'No schedules found' : 'Loading...'}
```

Comment:
Shouldn't this check for NPE? `getSearchResult` seems to return undefined for one case